clients/admin-ui/src/features/config-wizard/DescribeSystemsForm.tsx

```diff
@@ -0,0 +1,233 @@
+        handleChangeStep(5);
+      }
+    } else {
+      // @ts-ignore
+      const { error: updateSystemError } = await updateSystem(systemBody);
+
+      if (updateSystemError) {
+        toast({
+          status: "error",
+          description: "Updating system failed.",
+        });
+      } else {
+        toast.closeAll();
+        handleChangeStep(5);
+      }
+    }
+
+    setIsLoading(false);
+  };
+
+  return (
+    <Formik
+      initialValues={initialValues}
+      enableReinitialize
+      onSubmit={handleSubmit}
+    >
+      {({ values }) => (
+        <Form>
+          <Stack ml="100px" spacing={10}>
+            <HorizontalStepper activeStep={1} steps={HORIZONTAL_STEPS} />
+            <Heading as="h3" size="lg">
+              {/* TODO FUTURE: Path when describing system from infra scanning */}
+              Describe your system
+            </Heading>
+            <div>
+              By providing a small amount of additional context for each system
+              we can make reporting and understanding our tech stack much easier
+              for everyone from engineering to legal teams. So let’s do this
+              now.
+            </div>
+            <Stack>
+              <Stack direction="row">
+                <CustomTextInput name="name" label="System name" />
+                <Tooltip
+                  fontSize="md"
+                  label="Give the system a unique, and relevant name for reporting purposes. e.g. “Email Data Warehouse”"
+                  placement="right"
+                >
+                  <QuestionIcon boxSize={5} color="gray.400" />
+                </Tooltip>
+              </Stack>
+
+              <Stack direction="row" mb={5}>
+                <CustomTextInput name="key" label="System key" />
+                <Tooltip
+                  fontSize="md"
+                  label="System key’s are automatically generated from the resource id and system name to provide a unique key for identifying systems in the registry."
+                  placement="right"
+                >
+                  <QuestionIcon boxSize={5} color="gray.400" />
+                </Tooltip>
+              </Stack>
+
+              <Stack direction="row" mb={5}>
+                <CustomTextInput
+                  name="description"
+                  label="System description"
+                />
+                <Tooltip
+                  fontSize="md"
+                  label="If you wish you can provide a description which better explains the purpose of this system."
+                  placement="right"
+                >
+                  <QuestionIcon boxSize={5} color="gray.400" />
+                </Tooltip>
+              </Stack>
+
+              <Stack direction="row" mb={5}>
+                <CustomCreatableSingleSelect
+                  isClearable
+                  id="system_type"
+                  label="System Type"
+                  name="system_type"
+                  options={
```

Comment:
this begs the product/design question, if this field only ever has one or zero creatable fields as options, why is it a select? why not just an input box? or am I missing something?

Reply:
There's more context [here](https://www.figma.com/file/NeFvOOb1RT2fcdqZT0vBF2?node-id=77:5892#209896430) for why this was requested 👀